docs/operator-manual/high_availability.md

```diff
@@ -371,3 +371,16 @@ Not all HTTP responses are eligible for retries. The following conditions will n
 
 * Responses with a status code indicating client errors (4xx) except for 429 Too Many Requests.
 * Responses with the status code 501 Not Implemented.
+
+
+## CPU/Memory Profiling
+
+Argo CD optionally exposes a profiling endpoint that can be used to profile the CPU and memory usage of the Argo CD component.
+The profiling endpoint is available on metrics port of each component. See [metrics](./metrics.md) for more information about the port.
```

Comment:
I do think the docs from previous PRs are quite a bit more detailed and specific with directions and examples, but that can be modified and added to in future PRs.

Reply:
@crenshaw-dev suggested using `argocd-cmd-params-cm` ConfigMap to make this new setting more consistent with other settings. I've just applied this change. Let me know if it looks better please.

Reply:
That definitely feels more friendly/better UX. I can't speak to consistency in CD though. It also doesn't impact the docs much, as that's largely orthogonal to the implementation. But again, docs can be done in future PRs and are not as important as the disable